test/functionalTest/cases/0501_cors/max_age.test

```diff
@@ -41,6 +41,7 @@ HTTP/1.1 200 OK
 Content-Length: 0
 Access-Control-Max-Age: 600
 Access-Control-Allow-Headers: REGEX(.*)
+Access-Control-Expose-Headers: REGEX(.*)
```

Comment:
As far as I see, the list is explicit (`define CORS_EXPOSED_HEADERS FIWARE_CORRELATOR", "FIWARE_TOTAL_COUNT", "RESOURCE_LOCATION`). Thus, I'd suggest to use the actual headers instead a generic regex, in order to make the test stronger.

Reply:
I fact.. would it make sense to to the same for `Access-Control-Allow-Headers`, I mean, to pass from REGEX(.*) to the actual list of headers?

Reply:
@fgalan, thank you for this comment. This made me realize I have skipped an important step in testing. Let me explain : )

My initial idea was to have a separate test file focusing on each CORS header and avoid failing of a specific test just because another CORS header has a wrong value.

For example, here in `max_age.test` we are focusing on the Access-Control-Max-Age header and this test should not fail in case there is something wrong with `Access-Control-Allow-Headers`. A specific test or a set of tests should be focusing on that header.

The reality is, there are no tests focusing on `Access-Control-Allow-Headers` or `Access-Control-Expose-Headers`. I'll go ahead and add them quickly to the set of CORS tests. Please check if the tests make sense as a whole then and we can discuss if we need any further improvements.

Reply:
Let me check if I'm undersetanding your point before providing feedback

You mean having a .test focusing on each headers, thus:

* In the first .test Access-Control-Max-Age: has particular values but Access-Control-Allow-Headers and Access-Control-Expose-Headers use REGEX(.*)
* In the second .test Access-Control-Allow-Headers: has particular values but Access-Control-Max-Age and Access-Control-Expose-Headers use REGEX(.*)
* In the third .test Access-Control-Expose-Headers: has particular values but Access-Control-Max-Age and Access-Control-Allow-Headers use REGEX(.*)

Is this correct?

Reply:
Yes that's the idea. We also have the Access-Control-Allow-Methods and Access-Control-Allow-Origin headers but they already have their own dedicated test files.

Reply:
The general rule we use in the .test is try to be as specific as possible, that is, to avoid the usage of `REGEX()` (and specially, `REGEX(.*)`) has much as possible.

Having said that, I think that in this particular case it makes sense to do an exception to the general rule and proceed as you suggest. However, I'd suggest to use the following in the preamble of the involved .test in order to be clear:

> This test is focused in header Access-Control-Max-Age. Thus, other CORS related headers use REGEX(.*) as value. However, note that those specific headers are tested with specific headers values in other .test files in this same directory

or something similar

Reply:
Fixed in 3e6e50a